docs/troubleshooting.md

```diff
@@ -0,0 +1,55 @@
+# Troubleshooting Zappr
+
+This page tries to explain common sources of error and give instructions how to fix them.
+
+Terminology:
+
+* Zapprfile: A file named `.zappr.yaml` located in the root directory of your repository.
+
+## I can't enable or disable checks for my repository
+
+![A Github 404 API error is visible](img/troubleshooting/no-enable-disable.png)
+
+You don't have admin access for this repository. Either become an admin yourself or ask one to toggle this check.
+ 
+## I updated my Zapprfile in a PR, but it's not picked up.
+
+Zappr reads your Zapprfile from the repository's default branch (if you're not sure it's `master`). Changes on a feature branch are ignored.
```

Comment:
On reading the sentence for the first time: "If you're not sure it's `master`" - then what? (I had to re-read to understand that "it's `master`" is the consequence.) How about rephrasing like "it's `master` by default"?